src/onefetch/info.rs

```diff
@@ -222,6 +232,7 @@ impl Info {
         let last_change = Info::get_date_of_last_commit(&git_history);
         let project_license = Detector::new()?.get_project_license(workdir_str);
         let dominant_language = Language::get_dominant_language(&languages_stats);
+        let dependencies = deps::Detector::new().get_deps_info(workdir_str).unwrap();
```

Comment:
This should probably be caught instead of unwrapped.

Reply:
would that look like
```rust
let dependencies = deps::Detector::new().get_deps_info(workdir_str)?;
```

Reply:
d60ecfe